.docker/ci/Dockerfile

```diff
@@ -8,31 +8,42 @@ ENV TERM xterm
 
 # Setup catkin workspace
 ENV CATKIN_WS=/root/ws_moveit
-RUN mkdir -p $CATKIN_WS/src
-WORKDIR $CATKIN_WS/src
+WORKDIR $CATKIN_WS
 
 # Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
-RUN wstool init . && \
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
+RUN \
+    mkdir src && \
+    cd src && \
+
     # Download moveit source so that we can get necessary dependencies
+    wstool init . && \
     wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
     wstool update && \
-    # Update apt because previous images clear this cache
+
```

Comment:
I like to have these newline, as they nicely group individual commands.

Reply:
adding spaces causes warnings in Docker as it will no longer be supported in future versions of docker, please revert

Reply:
```
docker build -t test1 .
```

Output:
```
[WARNING]: Empty continuation line found in:
    RUN     mkdir src &&     cd src &&     wstool init . &&     wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall &&     wstool update &&     apt-get -qq update &&     apt-get -qq dist-upgrade &&     apt-get -qq install -y         wget         sudo         python-catkin-tools         ccache &&     rosdep update &&     rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false &&     cd .. &&     rm -rf src/ &&     rm -rf /var/lib/apt/lists/*
[WARNING]: Empty continuation lines will become errors in a future release.
```

One related [reference](https://stackoverflow.com/questions/46708053/empty-continuation-lines-will-become-errors-how-should-i-comment-my-dockerfil) article

Reply:
Thanks. I was using the default docker shipped with Xenial, which is 1.13 instead of 1.17.
To keep some grouping, I replaced the empty lines with an empty comment line now.